app/helpers/users_helper.rb

```diff
@@ -0,0 +1,9 @@
+module UsersHelper
+  def admin? user
+    user.admin?
+  end
+
+  def admin
+    @admin = User.find_by admin: true
+  end
```

Comment:
Này chắc mình đổi tên thành admin email rồi return email của admin thôi cho an toàn ông

Reply:
à khúc này tôi lấy admin để truyền tên admin vào cái mail gửi về nữa ông